src/DBAL/QueryFunction.php

```diff
@@ -0,0 +1,357 @@
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function replace(string|QueryExpression $expression, string|QueryExpression $search, string|QueryExpression $replace, ?string $alias = null): QueryExpression
+    {
+        return self::getExpression('REPLACE', [$expression, $search, $replace], $alias);
+    }
+
+    /**
+     * Build a FROM_UNIXTIME SQL function call
+     * @param string|QueryExpression $expression Expression to convert
+     * @param string|QueryExpression|null $format Function result alias
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function fromUnixTimestamp(string|QueryExpression $expression, string|QueryExpression $format = null, ?string $alias = null): QueryExpression
+    {
+        $params = [$expression];
+        if ($format !== null) {
+            $params[] = $format;
+        }
+        return self::getExpression('FROM_UNIXTIME', $params, $alias);
+    }
+
+    /**
+     * Build a DATE_FORMAT SQL function call
+     * @param string|QueryExpression $expression Expression to format
+     * @param string|QueryExpression $format Format to use
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function dateFormat(string|QueryExpression $expression, string|QueryExpression $format, ?string $alias = null): QueryExpression
+    {
+        return self::getExpression('DATE_FORMAT', [$expression, $format], $alias);
+    }
+
+    /**
+     * Build a LPAD SQL function call
+     * @param string|QueryExpression $expression Expression to pad
+     * @param string|QueryExpression $length Length to pad to
+     * @param string|QueryExpression $pad_string String to pad with
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function lpad(string|QueryExpression $expression, string|QueryExpression $length, string|QueryExpression $pad_string, ?string $alias = null): QueryExpression
+    {
+        return self::getExpression('LPAD', [$expression, $length, $pad_string], $alias);
+    }
+
+    /**
+     * Buidl a ROUND SQL function call
+     * @param string|QueryExpression $expression Expression to round
+     * @param string|QueryExpression|null $precision Precision to round to
+     * @param string|null $alias Function result alias (will be automatically quoted)
+     * @return QueryExpression
+     */
+    public static function round(string|QueryExpression $expression, string|QueryExpression $precision = null, ?string $alias = null): QueryExpression
+    {
+        if ($precision === null) {
+            $precision = new QueryExpression('0');
+        }
+        return self::getExpression('ROUND', [$expression, $precision], $alias);
+    }
```

Comment:
Precision will always be an integer here.

```suggestion
    public static function round(string|QueryExpression $expression, int $precision = 0, ?string $alias = null): QueryExpression
    {
        global $DB;
        $precision = new QueryExpression((string)$precision);
        return self::getExpression('ROUND', [$expression, $precision], $alias);
    }
```